server/controllers/submission.js

```diff
@@ -74,7 +74,6 @@ exports.createSubmission = asyncHandler(async (req, res, next) => {
 });
 
 exports.getAllSubmissions = asyncHandler(async (req, res, next) => {
-    console.log(req.params);
     const contest_id = req.params.id;
     const Creator = req.params.creator;
     console.log("contest id",contest_id);
```

Comment:
You can remove this console log too

Reply:
Yeah I will remove that right away, that one slipped away from me!